sklearn/gaussian_process/kernels.py

```diff
@@ -1616,6 +1627,183 @@ def __repr__(self):
             self.__class__.__name__, self.length_scale, self.periodicity)
 
 
+class LinearKernel(Kernel):
+    """Linear kernel.
+
+    The Linear kernel is non-stationary and can be obtained from Bayesian
+    linear regression by putting N(0, \sigma_v^2) priors on the coefficients of
+    x_d (d = 1, . . . , D) and a prior of N(0, \sigma_b^2) on the bias, with
+    the origin being shifted by c. It is parameterized by parameters sigma_v,
+    sigma_0 and c. The parameters of the Linear kernel are about specifying the
+    origin. The kernel is given by:
+
+    k(x_i, x_j) = sigma_b ^ 2 + sigma_v ^ 2 * ((x_i - c) \cdot (x_j - c))
```

Comment:
In any case, there is no need for `sigma_b` and `sigma_v` as hyperparameters. These can be achieved by doing something similar to `ConstantKernel() +ConstantKernel() * LinearKernel() `